crates/polars-core/src/frame/column/mod.rs

```diff
@@ -1018,15 +1028,64 @@ impl Column {
     }
 
     pub fn bitand(&self, rhs: &Self) -> PolarsResult<Self> {
+        // @partition-opt
+        // @scalar-opt
         self.as_materialized_series()
             .bitand(rhs.as_materialized_series())
             .map(Column::from)
     }
+    pub fn bitor(&self, rhs: &Self) -> PolarsResult<Self> {
```

Comment:
I don't really understand why we have these and std::ops::BitAnd, etc. on Series.

Reply:
Me neither. Maybe we can try to remove it in a follow op.